src/hooks/useConnectPartyGoers.ts

```diff
@@ -1,25 +0,0 @@
-import { useFirestoreConnect } from "react-redux-firebase";
```

Comment:
I have looked through our hooks and thought that it would be nice to have all `users-related` hooks in `users.ts`. Mostly, style preference, but my intent was to also make `useUsersConnect` private hook, having it non-exportable in `users.ts`. We have a somewhat similar approach in `roles.ts`. With application growth, I think we will have more hooks onboard and it's easier to keep track of them by grouping them in such files. WDYT?

Reply:
I will mark this dialog as `hooks grouping discussion` and will use that to answer similar comments

Reply:
That makes sense RE: making `useUsersConnect` private + keeping all of the `users-related` hooks together 👍🏻 

I'm happy to keep them together in these 'themed' files like you have done here as a pattern when they are quite short/simple hooks. Where it doesn't make sense is with longer/more complex code, where the file can get way too long and confusing/hard to maintain. Another alternative to putting them all in 1 file would be to use subfolders, so perhaps `hooks/users/useUserFoo.ts`, that way we can use the same pattern for all of them regardless of size/complexity. For now I think they are all simple enough (the file is only 70 lines) that they can stay in the one file. But we should keep this in the back of our minds in future if things get longer/more complex/etc.

Not that it is directly relevant to this PR, but I also tend to have a fairly strong preference for not collating multiple components into the same code file, unless they are super tiny/simple sub-components that will only ever be used/needed within the 'main' component in the file. I've had to deal with some monstrously huge/complex/confusing files in my past, where they would have been far simpler if they just stuck to 1 react component per file.